apps/gitness/src/framework/context/AppContext.tsx

```diff
@@ -1,15 +1,21 @@
-import React, { createContext, useContext, useState, ReactNode } from 'react'
+import React, { createContext, useContext, useState, ReactNode, useEffect } from 'react'
 import {
   CodeServiceAPIClient,
   TypesMembershipSpace,
   membershipSpaces,
-  TypesSpace
+  TypesSpace,
+  TypesUser,
+  getUser
 } from '@harnessio/code-service-client'
+import { useAtom } from 'jotai'
```

Comment:
Lets add a SecOps ticket for this.

Reply:
sure will add a secops ticket

Reply:
What does `useAtom` do?

Reply:
Useatom is just a more flexible state management that can update particular states of a object and was used in diffs and PRs and then tan adopted it to other places @vardanbansal-harness

Reply:
So it's like a reducer right? @srdjan-harness has already added one here https://github.com/harness/canary/blob/main/apps/gitness/src/hooks/useThunkReducer.tsx

Can we just try to use it? Two different reducers in a single app is not a good idea.